### PR TITLE
[TACHYON-693] Update the docs/Command-Line-Interface.md file (add the…

### PR DESCRIPTION
**Pull request by tianyin**
… docs for du, getUsedBytes, and, getCapacityBytes)

### CODE REVIEW

**Comment by calvinjia** [comment]
@tianyin Thanks for the patch, could you order the commands in alphabetical order?

**Comment by calvinjia** [comment]
+1 LGTM

File: docs/Command-Line-Interface.md [hunk 1]
```diff
@@ -28,11 +28,51 @@ Or, if no header is provided, the default hostname and port (set in the env file
...
+  </tr>
+  <tr>
     <td>count</td>
     <td>count "path"</td>
     <td>Display the number of folders and files matching the specified prefix in "path".</td>
   </tr>
   <tr>
+    <td>du</td>
+    <td>du "path"</td>
+    <td>Displays the size of a file or a directory specified by the input path.</td>
```
Comment (apc999) [inline docs/Command-Line-Interface.md]: Displays -> Display, to keep consistent?

**Comment by apc999** [comment]
one minor wording suggestion left.
Thanks for the patch!

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**